src/app/components/Topbar/styles.module.scss

```
@@ -77,13 +77,21 @@ $shadow: 0 0 7px 0 rgba(122, 122, 122, 0.81);
   min-width: 80px !important;
 
   @media #{$small-mobile} {
+    font-size: $small;
     margin: 0px 1px;
-    padding: 3px 8px
+    padding: 3px 8px;
+    width: 15%;
   }
 }
 
 .topBarButton:disabled {
   color: $white !important;
+  @media #{$small-mobile} {
+    font-size: $small;
+    margin: 0px 1px;
```

There is no need to change this now, but you should know that linter wouldn't let you commit when having a 0 value with the unit defined
```suggestion
    margin: 0 1px;
```